cluster/gce/gci/configure-helper.sh

```diff
@@ -721,14 +721,14 @@ function start-kube-addons {
     local -r file_dir="cluster-monitoring/${ENABLE_CLUSTER_MONITORING}"
     setup-addon-manifests "addons" "${file_dir}"
     # Replace the salt configurations with variable values.
-    base_metrics_memory="200Mi"
+    base_metrics_memory="140Mi"
     metrics_memory="${base_metrics_memory}"
-    base_eventer_memory="200Mi"
+    base_eventer_memory="190Mi"
     eventer_memory="${base_eventer_memory}"
     local -r metrics_memory_per_node="4"
     local -r eventer_memory_per_node="500"
     if [[ -n "${NUM_NODES:-}" && "${NUM_NODES}" -ge 1 ]]; then
-      num_kube_nodes="$((${NUM_NODES}-1))"
+      num_kube_nodes="$((${NUM_NODES}+1))"
```

Comment:
When I made the code change for GCI config, I simply followed the logic in the corresponding salt config file (e.g., https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml). I assumed that logic there is correct, so I did not have to spend tons of time on understanding every piece of details. If you notice the logic in the salt-based config changes, please feel free to change GCI config. Thanks